doc/sphinx/addendum/sprop.rst

```diff
@@ -241,15 +241,33 @@ The term :g:`c (top -> top) (fun x => x) c` infinitely reduces to itself.
 Lack of tactic support
 ----------------------
 
-Some tactics do not handle |SProp| as they were not correctly ported. While in
-most of the cases they will just fail, in some cases this can result in sending
-ill-typed terms to the kernel, which will fail with anomalies at `Qed`.
+Every binder in a term (such as `fun x` or `forall x`) caches
+information called the :gdef:`relevance mark` indicating whether its type is
+in |SProp| or not. This is used to efficiently implement proof
+irrelevance.
+
+Relevance marks are invisible to the user, but code outside the kernel
+may generate incorrect marks resulting in bugs. Typically this means a
+conversion will incorrectly fail as a variable was incorrectly marked
+proof relevant.
 
```

Comment:
```suggestion
Relevance marks are shown as comments embedded in binders in a :n:`term`, which can be `(* Relevant *)`, `(* Irrelevant *)` or `(* <ADD WORDS HERE> *).
```

Reply:
this is what the example is for, no need to specify it